src/redemptions/redemption.service.ts

```diff
@@ -312,18 +320,6 @@ export class RedemptionService {
       };
     }
 
-    if (
-      this.hasOnlyBenignWarnings(labels) &&
-      transactionStatus.decision.risk.score < 50
```

Comment:
Had to remove acceptable risk score to allow these passes. Note that scammers should be caught by line 273 in this file.